src/legacy/ui/public/field_editor/field_editor.js

```diff
@@ -727,7 +726,7 @@ export class FieldEditor extends PureComponent {
     if (!fieldFormatId) {
       indexPattern.fieldFormatMap[field.name] = undefined;
     } else {
-      indexPattern.fieldFormatMap[field.name] = fieldFormat;
```

Comment:
regression of #48108.

`indexPattern.fieldFormatMap[field.name] = field.format;`  - was right